scss/README.md

```diff
@@ -38,3 +38,77 @@ $mobile-ie6: false;
 
 @import "index.scss";
 ```
+
+## Styleguide
+
+### Naming
+
+The naming of classes and ids normally follows one of these patterns:
+
```

Comment:
Should we make it explicit that the simplest pattern is just **thing**, e.g. `button`, `question`?